CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## Unreleased
+
+* Allow links in table headers with a scope of row
```

Comment:
I think this should match the PR title and include the PR number, as per previous entries.

Reply:
Will fix.